packages/protocol/scripts/foundry/constants.sh

```diff
@@ -5,6 +5,7 @@ export FROM_ACCOUNT="0x$FROM_ACCOUNT_NO_ZERO" # Anvil default account (1)
 # Anvil configurations (Source: https://book.getfoundry.sh/reference/anvil/)
 export ANVIL_PORT=8546
 export ANVIL_RPC_URL="http://127.0.0.1:$ANVIL_PORT"
+export ANVIL_LOGGING_ENABLED=${ANVIL_LOGGING:=true} # Flag to enable or disable logging. Useful for local development
```

Comment:
It would probably make sense to have it disabled as default (even though it was enabled up until now)